city_scrapers/spiders/chi_state_university.py

```diff
@@ -0,0 +1,151 @@
+import re
+from calendar import month_name
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD, COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiStateUniversitySpider(CityScrapersSpider):
+    name = "chi_state_university"
+    agency = "Chicago State University"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.csu.edu/boardoftrustees/dates.htm"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+
+        for data in response.xpath("/html/body/div/div[4]/div/div[1]/div/div"):
+            titles = data.xpath(".//label/text()").extract()
+
+            for i, tab in enumerate(data.xpath('.//div[@class = "itd-tab"]')):
+
+                for item in tab.xpath(".//li"):
+
+                    months = {m.lower() for m in month_name[1:]}
+                    text = item.xpath(".//text()").extract()[0]
+                    monthMatch = next(
+                        (word for word in text.split() if word.lower() in months), None
+                    )
+                    if monthMatch is None:
+                        continue
+
+                    meeting = Meeting(
+                        title=titles[i],
+                        description=self._parse_description(item),
+                        classification=self._parse_classification(i),
+                        start=self._parse_start(item),
+                        end=None,
+                        all_day=False,
+                        time_notes=self._parse_time_notes(item),
+                        location=self._parse_location(item),
+                        links=self._parse_links(item),
+                        source=self._parse_source(response),
+                    )
+                    meeting["status"] = self._get_status(meeting)
+                    meeting["id"] = self._get_id(meeting)
+
+                    yield meeting
+
+    def _parse_description(self, item):
+        parts = item.xpath(".//text()").extract()
+        description = "".join([str(x) for x in parts])
+        description = (
+            description.replace("\xa0", " ")
+            .replace("\n", " ")
+            .replace("                                     ", " ")
+            .replace("\t", " ")
+            .strip()
+        )
+        return description
+
+    def _parse_classification(self, i):
```

Comment:
It might be a little more transparent if we check the title instead of the tab index in case they change that